src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleErrorFieldSymbol.vb

```diff
@@ -0,0 +1,100 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Globalization
+Imports System.Threading
+Imports Microsoft.CodeAnalysis
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    ''' <summary>
+    ''' Represents a field of a tuple type (such as (int, byte).Item1)
+    ''' that doesn't have a corresponding backing field within the tuple underlying type.
+    ''' Created in response to an error condition.
+    ''' </summary>
+    Friend Class TupleErrorFieldSymbol
+        Inherits SynthesizedFieldSymbol
+
+        ''' <summary>
+        ''' If this field represents a tuple element (including the name match), 
+        ''' id is an index of the element (zero-based).
+        ''' Otherwise, (-1 - [index in members array]);
+        ''' </summary>
+        Private _tupleFieldId As Integer
+
+        Private _locations As ImmutableArray(Of Location)
+
+        Private _useSiteDiagnosticInfo As DiagnosticInfo
+
+        Public Overrides ReadOnly Property IsTupleField() As Boolean
```

Comment:
fixed

---
In reply to: [71545531](https://github.com/dotnet/roslyn/pull/12596#discussion_r71545531) [](ancestors = 71545531)